README.md

```diff
@@ -48,63 +59,146 @@ python scripts/make.py "qt/bin" --lib-path "nomacs/3rd-party/build"
+- [kimageformats-plugins]([https://github.com/KDE/kimageformats](https://invent.kde.org/frameworks/kimageformats)) is an optional dependency that provides additional formats such as AVIF, HEIC/HEIF, and JPEG XL/JXL. The Qt version of the plugins should match the Qt version when compiling nomacs.
+- Zip file support requires Quazip, which has varied support in distributions. If the system package is missing or does not work, you can set `USE_SYSTEM_QUAZIP=NO` to use the version in nomacs/3rdparty. However, you may also need to remove the any system quazip development package temporarily. (such as `llibquazip*-dev` for Ubuntu)
 
-  ```console
-  sudo apt-get install debhelper cdbs qt5-qmake qttools5-dev-tools qtbase5-dev qttools5-dev libqt5svg5-dev qt5-image-formats-plugins libexiv2-dev libraw-dev libopencv-dev cmake libtiff-dev libquazip5-dev libwebp-dev git build-essential lcov libzip-dev
-  ```
+### Get the required packages
 
-- For older Ubuntu versions:
+The package lists and cmake configuration listed below enable all features in nomacs, which we recommend. However, if you do not want a particular feature you can omit certain packages.
 
-  ```console
-  sudo apt-get install debhelper cdbs qt5-qmake qttools5-dev-tools qt5-default qttools5-dev libqt5svg5-dev qt5-image-formats-plugins libexiv2-dev libraw-dev libopencv-dev cmake libtiff-dev libquazip5-dev libwebp-dev git build-essential lcov libzip-dev
-  ```
+This is the list of configurable packages and what they provide in nomacs. If you want to be sure a feature is included/excluded set the cmake option for that feature. By default, all features are enabled if the dependencies are met**.
 
-Clone the nomacs repository from GitHub:
+| Package | Required | Depends | cmake | Description
+| -       | -        | -       | -     | -
+| Qt      |  Yes     |         | `QT_VERSION_MAJOR` | Qt components: Core, Concurrent, Network, PrintSupport, SVG, Widgets, Core5Compat
+| LibRAW  |  No      | OpenCV  | `ENABLE_RAW`       | Enables reading RAW images
+| LibTiff |  No      | OpenCV  | `ENABLE_TIFF`      | Enable reading multi-page TIFF
+| Quazip**|  No      |         | `ENABLE_QUAZIP`    | Enable reading from zip files
+| OpenCV  |  No      |         | `ENABLE_OPENCV`    | RAW, TIFF, Adjustments, High-quality thumbnails, DRIF files, histogram, mosaic
+| nomacs/plugins| No | OpenCV  | `ENABLE_PLUGINS`   | Paint on image, Composite, Affine Transform, Fake Miniatures, Page Extraction
+| googletest    | No |         | `ENABLE_TESTING`   | Enables `make check` target for unit testing
+| QImageFormats | No |         | n/a                | Enables reading ICNS, MNG, TGA, TIFF, WBMP, WEBP
+| KImageFormats | No |         | n/a                | Enables reading AVIF, HEIF/HEIC, JXL, EXR, EPS and [more](https://github.com/KDE/kimageformats)
+
+(**) Quazip is not enabled by default as of nomacs 3.19.1, this may change in a future update.
```

Comment:
I think this is a breaking change and we should not do this unless necessary.

Reply:
I assume you mean making quazip default enabled. I'm not advocating that we do this anytime soon,  but  I do not think it would break much and the trend seems to be always enabling it.

d7e555f878597fdb was the last to disable quazip (it has gone back and forth a few times) , but it is not clear to me why  (@novomesk  might remember why). Since then we have updated 3rdparty/quazip.

- It is a somewhat unique feature, I believe one significant reason users select nomacs over other viewers
- Quazip1 seems to be present in all distros now
- Most maintainers  seem to be setting ENABLE_QUAZIP=ON already (they were on 3.17 too). 
- Quazip < 0.7.6 had a CVE which we resolved by updating the submodule
- We fixed the build so  USE_SYSTEM_QUAZIP works on everything again